src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -872,6 +877,18 @@ public static function findByOptions($options)
                 ->getTracks()
             );
     }
+    /**
+     * This method allows direct access to iTonesPlaylists, without all the weird stuff in search api.$_COOKIE
+     *
+     * @param int itonesplaylistid The id of the itones playlist we want to get tracks for.
+    */
+    public static function itonesPlaylist($itonesplaylistid = null) {
```

Comment:
If the intention is for an API endpoint, why not just expose iTones_Playlist->getTracks()?